src/component/Button.js

```diff
@@ -1,14 +1,30 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import '../index.scss';
 
-const Button = ({ name }) => <div>{name}</div>;
+const Button = ({ name, wide, color }) => {
+  const buttonPropStyle = {
+    width: `${wide ? '50%' : '25%'}`,
+    backgroundColor: color,
+  };
+
+  return (
+    <div className="button" style={buttonPropStyle}>
```

Comment:
- [ ] Should we replace this `div` with a `button` element instead?

Reply:
Thanks for your suggestion. It is updated.